plugins/source/aws/resources/services/elasticache/gen.hcl

```diff
@@ -123,4 +123,577 @@ resource "aws" "elasticache" "clusters" {
   column "pending_modified_values_log_delivery_configurations" {
     skip = true
   }
+}
+
+resource "aws" "elasticache" "engine_versions" {
+  path = "github.com/aws/aws-sdk-go-v2/service/elasticache/types.CacheEngineVersion"
+
```

Comment:
Could we add a description to this table?

Reply:
The description is autofilled by cq-gen :)

Reply:
Ah okay, nice 😄